modules/gapi/test/common/gapi_tests_common.hpp

```diff
@@ -196,25 +196,32 @@ class TestFunctional
         }
     }
 
+    void initMatsFromImages(int channels, const std::string& pattern, int imgNum)
+    {
+        initTestDataPath();
+        GAPI_Assert(channels == 1 || channels == 3 || channels == 4);
+        const int flags = (channels == 1) ? cv::IMREAD_GRAYSCALE : cv::IMREAD_COLOR;
+
+        cv::Mat m1 = cv::imread(findDataFile(cv::format(pattern.c_str(), imgNum)), flags);
+        cv::Mat m2 = cv::imread(findDataFile(cv::format(pattern.c_str(), imgNum + 1)), flags);
```

Comment:
Avoid "cv::" prefixes in OpenCV code (if they are not required).

Reply:
Is it a general rule? I thought this increases readability..